screens/ChallengesScreen.js

```diff
@@ -1,52 +1,83 @@
 import * as WebBrowser from "expo-web-browser";
 import React from "react";
+import CardFlip from "react-native-card-flip";
+import { MapView } from "expo";
+import FlipComponent from "react-native-flip-component";
+import ChallengeCard from "../components/ChallengeCard";
+import RevealChallenge from "../components/RevealChallenge";
+import KitText from "../components/KitText";
+import Colors from "../constants/Colors";
 import {
-  Button,
-  Image,
-  Platform,
-  ScrollView,
   StyleSheet,
   Text,
   TextInput,
   TouchableOpacity,
   View
 } from "react-native";
 
-import signInWithFacebook from "../utils/auth/auth";
-import  KitText from "../components/KitText";
+export default function ChallengesScreen() {
+  // constructor = props => {
```

Comment:
The flip might not be working because this is a functional component instead of a class component. You can either keep the functional component and figure out how to do the flip with hooks (to match the rest of the app) or just turn this into a class component (to match the example that I assume you're following).